marketplace-service/src/main/java/com/axonivy/market/util/VersionUtils.java

```diff
@@ -145,4 +163,12 @@ public static List<String> getReleaseTagsFromProduct(Product product) {
     return product.getReleasedVersions().stream().map(
         version -> convertVersionToTag(product.getId(), version)).toList();
   }
+
+  public static List<String> getNonSyncedVersionOfTagsFromMetadataSync(List<String> releasedVersion,
+      MetadataSync cache) {
+    if (!CollectionUtils.isEmpty(cache.getSyncedTags())) {
```

Comment:
this method only use getSyncedTags() of MetadataSync , we should not pass the whole object to function but then just use one